api/v1/flow/flow.proto

```diff
@@ -306,8 +306,7 @@ enum DebugCapturePoint {
     DBG_CAPTURE_FROM_LB = 5;
     DBG_CAPTURE_AFTER_V46 = 6;
     DBG_CAPTURE_AFTER_V64 = 7;
-    DBG_CAPTURE_PROXY_PRE = 8;
-    DBG_CAPTURE_PROXY_POST = 9;
+    // Indices 8 and 9 are deprecated
```

Comment:
I don't think we should remove them entirely as that will be a breaking change. Let's keep them but also move the deprecation comment above them to indicate as such.